platform/web/export/export_plugin.cpp

```diff
@@ -485,6 +485,7 @@ Error EditorExportPlatformWeb::export_project(const Ref<EditorExportPreset> &p_p
 	}
 	html.resize(f->get_length());
 	f->get_buffer(html.ptrw(), html.size());
+	f.unref(); // close file.
```

Comment:
Can `html_path` be the same as `p_path`? It seems to be reading and writing a different files.

Reply:
It changes if use a custom html shell